### PR TITLE
Italian translation

### LINKED ISSUE #3548: Translate to Italian
Guide to translating:
https://github.com/MetaMask/metamask-extension/blob/master/docs/translating-guide.md

### PR DESCRIPTION
**Pull request by matteopey**
Fixes #3548
I will ask to some italian people to review. :)

### CODE REVIEW

File: app/_locales/it/messages.json [hunk 1]
```diff
@@ -0,0 +1,609 @@
...
+    "message": "Vedi Account"
+  },
+  "warning": {
+    "message": "Attenzione"
+  },
+  "whatsThis": {
+    "message": "Cos'è questo?"
+  },
+  "yourSigRequested": {
+    "message": "La tua firma sta venendo richiesta"
```
Comment (fiblan) [inline app/_locales/it/messages.json]: "E' richiesta la tua firma" or "La tua firma è richiesta"

**Review by fiblan — CHANGES_REQUESTED** [review]
Just only one change

**Comment by tmashuang** [comment]
We have made added more to the `messages.json` file, please update and add.

**Review by kumavis — APPROVED** [review]

**Comment by kumavis** [comment]
@fiblan Please accept the invitation to the MetaMask translator team https://github.com/MetaMask

**Pull request merged**